### ISSUE TITLE
Param for assigning flashlights to PCs/NPCs

### ISSUE THREAD

**Issue by ninjaProtocol** [ferstaberinde/F3#748]
I suppose that  this might end up in AI assign gear anyway, but @Wolfenswan had this script to remove NVGs and laser pointers from the AI, and give them all flashlights. Was thinking if this could be made into a part of F3 as a param.
This script: http://www.folkarps.com/forum/viewtopic.php?f=48&t=1403

I modified this for a mission of mine to remove the NVGs (via param) for players, the example file is attached.

Param Example:
`// description.ext

	class sk_param_removePlayerNVG
	{
		title = "Player NVGs";
		values[] = {0,1};
		texts[] = {"Disabled", "Enabled"};
		default = 1;
		code = "sk_NVGenabled = %1";
	};`

//init.sqf
`[] execVM "removePlayerNVG.sqf";`

Example (can be extended to "players only", "AI only", "everyone"):
[param_flashlights.txt](https://github.com/ferstaberinde/F3/files/339094/param_flashlights.txt)

Note that in the example I'm excluding attack helo pilots - this is a mission specific thing that I needed.

**Comment by ferstaberinde** [comment]
@ninjaProtocol - I'd really like to include this in v3-4-1, but would like the parameter to simply control whether or not AI units get flashlights (we'll leave the equipment of player units out of scope).

**Comment by Wolfenswan** [comment]
I don't think this should be a parameter but down to the mission maker to enable/disable.

I also don't see why this needs a new component, as it can be done using assign gear AI.
